SignalBackgroundMerger.cpp

```diff
@@ -320,32 +353,72 @@ class SignalBackgroundMerger {
-    std::cout << "\t- " << signalFile << "\t" << freqTerm << "\n";
+    std::cout << "\t- " << signalFile << "\t" << freqTerm << "\n" << statusTerm << "\n";
     
     std::cout << "\nBackground files and their respective frequencies:\n";
     if (!bg1File.empty()) {
       freqTerm = bg1Freq > 0 ? std::to_string(bg1Freq) + " kHz" : "(from weights)";
-      std::cout << "\t- " << bg1File << "\t" << freqTerm << "\n";
+      statusTerm = bg1Status > 0 ? statusMessage + std::to_string(bg1Status) : "";
+      std::cout << "\t- " << bg1File << "\t" << freqTerm << "\n" << statusTerm << "\n";
+      if (bg1Status>0){
+        statusList_stable.push_back(bg1Status+1);
+        statusList_decay.push_back(bg1Status+2);
+      }
     }
     if (!bg2File.empty()) {
       freqTerm = bg2Freq > 0 ? std::to_string(bg2Freq) + " kHz" : "(from weights)";
-      std::cout << "\t- " << bg2File << "\t" << freqTerm << "\n";
+      statusTerm = bg2Status > 0 ? statusMessage + std::to_string(bg2Status) : "";
+      std::cout << "\t- " << bg2File << "\t" << freqTerm << "\n" << statusTerm << "\n";
+      if (bg2Status>0){
+        statusList_stable.push_back(bg2Status+1);
+        statusList_decay.push_back(bg2Status+2);
+      }
     }
     if (!bg3File.empty()) {
       freqTerm = bg3Freq > 0 ? std::to_string(bg3Freq) + " kHz" : "(from weights)";
-      std::cout << "\t- " << bg3File << "\t" << freqTerm << "\n";
+      statusTerm = bg3Status > 0 ? statusMessage + std::to_string(bg3Status) : "";
+      std::cout << "\t- " << bg3File << "\t" << freqTerm << "\n" << statusTerm << "\n";
+      if (bg3Status>0){
+        statusList_stable.push_back(bg3Status+1);
+        statusList_decay.push_back(bg3Status+2);
+      }
     }
     if (!bg4File.empty()) {
       freqTerm = bg4Freq > 0 ? std::to_string(bg4Freq) + " kHz" : "(from weights)";
-      std::cout << "\t- " << bg4File << "\t" << freqTerm << "\n";
+      statusTerm = bg4Status > 0 ? statusMessage + std::to_string(bg4Status) : "";
+      std::cout << "\t- " << bg4File << "\t" << freqTerm << "\n" << statusTerm << "\n";
+      if (bg4Status>0){
+        statusList_stable.push_back(bg4Status+1);
+        statusList_decay.push_back(bg4Status+2);
+      }
     }	
+     auto join = [](const std::vector<int>& vec) {
+        return std::accumulate(vec.begin(), vec.end(), std::string(),
+            [](const std::string& a, int b) {
+                return a.empty() ? std::to_string(b) : a + " " + std::to_string(b);
+            });
+    };
+    std::string stableStatuses = join(statusList_stable);
+    std::string decayStatuses = join(statusList_decay);
+    std::string message = "\n!!!Attention!!!\n To proceed the shifted particles statuses in DD4hep, please add the following options to ddsim:\n"
+                          "--physics.alternativeStableStatuses=\"" + stableStatuses + 
+                          "\"  --physics.alternativeDecayStatuses=\"" + decayStatuses + "\"\n";
```

Comment:
Does this need
```suggestion
                          "--physics.alternativeStableStatuses=\"{" + stableStatuses + 
                          "}\"  --physics.alternativeDecayStatuses=\"{" + decayStatuses + "}\"\n";
```
I don't know.